src/domain/safe/entities/schemas/module-transaction-type.schema.ts

```diff
@@ -24,6 +24,7 @@ export const moduleTransactionTypeSchema: Schema = {
     isSuccessful: { type: 'boolean' },
     transactionHash: { type: 'string' },
     module: { type: 'string' },
+    moduleTransactionId: { type: 'string' },
```

Comment:
This change belongs to the domain layer, and it was left behind in the domain changes PR (https://github.com/safe-global/safe-client-gateway-nest/pull/312/files). It might be moved to a separate PR, I'm open to moving it. But I decided to include it here for brevity and because is related to the mappings (in fact, the mappings depend on this change).